packages/augur-ui/src/modules/create-market/components/common.styles.less

```diff
@@ -212,7 +212,10 @@
 		.text-16-medium;
 
 		color: @color-alternate-info-text;
-	}
+  }
+  > span:last-of-type {
```

Comment:
this isn't right. it will effect all the other small subheaders

Reply:
what is the right way?

Reply:
do it in the individual time component or > span:nth-of-type(3) could work.